### PR TITLE
Bug 1877858: render ConsoleNotification BannerTop if notification.spec.…

### PR DESCRIPTION
**Pull request by rhamilto**
…location not specified

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1877858

### CODE REVIEW

File: frontend/public/components/console-notifier.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ const ConsoleNotifier_: React.FC<ConsoleNotifierProps> = ({ obj, location }) =>
     <>
       {_.map(_.get(obj, 'data'), (notification) =>
         notification.spec.location === location ||
-        notification.spec.location === 'BannerTopBottom' ? (
+        notification.spec.location === 'BannerTopBottom' ||
+        // notification.spec.location is optional, so always output the notification
+        // if location is not specified, producing the same result as location: BannerTopBottom
```
Comment (spadgett) [inline frontend/public/components/console-notifier.tsx:21]: I'm OK with this, but I have a slight preference for defaulting to `BannerTop`. I think that's usually what people want.

Comment (rhamilto) [reply frontend/public/components/console-notifier.tsx:21]: I went with this because it was the simplest code change...

Comment (rhamilto) [reply frontend/public/components/console-notifier.tsx]: Updated to default location to `BannerTop` if `location` not specified.

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**